lib/bloc/app_blocs.dart

```diff
@@ -23,14 +25,16 @@ class AppBlocs {
     POSProfileBloc posProfileBloc = new POSProfileBloc();    
     InvoiceBloc invoicesBloc = new InvoiceBloc();
     ConnectPayBloc connectPayBloc = new ConnectPayBloc(userProfileBloc.userStream, accountBloc.accountStream);
-    
+    BackupBloc backupBloc = new BackupBloc(accountBloc);
```

Comment:
The BackupBloc only needs the account stream so it is better to pass it instead of the accountBloc